ansible/playbooks/adhoc/docker_loopback_to_lvm/docker_loopback_to_direct_lvm.yml

```diff
@@ -0,0 +1,142 @@
+---
```

Comment:
I'd like to see this get moved to openshift-ansible-contrib rather than openshift-tools (or maybe both). This is something that we could use for our customers.